packages/validation-manager/src/ValidationManagerRIP7560.ts

```diff
@@ -25,6 +28,43 @@ export class ValidationManagerRIP7560 implements IValidationManager {
     // TODO
   }
 
+  async _getStakesInfo (operation: OperationBase): Promise<{ senderInfo: StakeInfo, paymasterInfo: StakeInfo, factoryInfo: StakeInfo }> {
```

Comment:
Given that this method may need to make up to 3 (or 4 if we add aggregators later) roundtrip calls to the `getDepositInfo` function, maybe it would make sense to modify the function itself to look something like this:
```
function getDepositInfo(address[] accounts) external view returns (DepositInfo[] memory infos);
```

In ERC-4337 we didn't need to because we made the validation frame `simulateValidation` function return the struct of stake infos, but I don't think we need to use "simulations" in RIP-7560.

Reply:
The whole point was to not make changes to the StakeManager contract..

Reply:
it is still a better API to request them all in one call.
1. we make multiple parallel calls to the node to request the value (using Promise.all)
2. we can later use a helper contract that "batch" multiple calls to the nonce manager and return multiple values (e.g. our "BundlerHelper") - I'm not saying to add it now, but it leaves a room for it.
The point is that if the API now is for a single address, it is harder to batch it later.

Reply:
Added